libraries/AP_HAL_SITL/SITL_State.cpp

```diff
@@ -311,7 +320,7 @@ void SITL_State::_fdm_input_local(void)
         sitl_model->fill_fdm(_sitl->state);
         _sitl->update_rate_hz = sitl_model->get_rate_hz();
 
-        if (_sitl->rc_fail == 0) {
+        if (_sitl->rc_fail != SITL::SITL::SITL_RCFail_NoPulses) {
             for (uint8_t i=0; i< _sitl->state.rcin_chan_count; i++) {
                 pwm_input[i] = 1000 + _sitl->state.rcin[i]*1000;
```

Comment:
No support for Throttle950 here? I think leaving this equal to 0 (I mean, using the enum) would actually be the correct thing - in `_check_rc_input` we set the appropriate values (or don't set anything at all) and here we would only override them if RC isn't failing.

Reply:
Changed this back to equivalence to not-failed.